Chessington.GameEngine/Pieces/Pawn.cs

```diff
@@ -13,17 +13,43 @@ public Pawn(Player player)
         public override IEnumerable<Square> GetAvailableMoves(Board board)
```

Comment:
This method is quite long now, might be worth trying to split out some smaller methods? 🙂

Reply:
I've broken this up in d063755, is this enough or do you think I should try to do some more 🙂